plugins/GSdx/GSState.cpp

```diff
@@ -46,7 +46,7 @@ GSState::GSState()
 	, m_frameskip(0)
 	, m_crcinited(false)
 {
-	m_nativeres = !!theApp.GetConfig("nativeres", 1);
+	m_nativeres = !!theApp.GetConfig("upscale_multiplier",1)==1 ? 1:0;
```

Comment:
that bool variable m_nativeres is used to check whether native resolution is used (or) not.  so there's a conditional check for whether upscale_multiplier == 1 (native resolution), if it's true, m_nativeres will be 1 and if it's false it will be 0. I don't understand what you mean by always 1 regardless of scaling ?